src/app/components/chat/user-online-status/AppChatUserOnlineStatus.vue

```diff
@@ -0,0 +1,117 @@
+<script lang="ts">
```

Comment:
uhhhh

Reply:
I'm assuming this means that you want me to move the import into the `setup` script tag. Made the change, but I'll leave this unresolved until I know that's what you wanted.